packages/engine/Source/Scene/Model/Model.js

```diff
@@ -342,10 +345,9 @@ function Model(options) {
   const scene = options.scene;
   if (defined(scene) && defined(scene.terrainProviderChanged)) {
     this._terrainProviderChangedCallback = scene.terrainProviderChanged.addEventListener(
-      function () {
+      () => {
```

Comment:
Question: do we now prefer ES6 arrow functions for new code? It's been a while since I worked on Cesium so maybe the style guidelines have changed?

Reply:
Currently we use both. It depends on the situation. Arrow functions preserve scope, which make then ideal for this situation as it avoid having to use `const that = this;`.